cmd/performance-profile-creator/cmd/root.go

```diff
@@ -311,6 +351,7 @@ func init() {
 	rootCmd.MarkPersistentFlagRequired("must-gather-dir-path")
 	rootCmd.PersistentFlags().StringVar(&args.ProfileName, "profile-name", "performance", "Name of the performance profile to be created")
 	rootCmd.PersistentFlags().StringVar(&args.TMPolicy, "topology-manager-policy", kubeletconfig.RestrictedTopologyManagerPolicy, fmt.Sprintf("Kubelet Topology Manager Policy of the performance profile to be created. [Valid values: %s, %s, %s]", kubeletconfig.SingleNumaNodeTopologyManagerPolicy, kubeletconfig.BestEffortTopologyManagerPolicy, kubeletconfig.RestrictedTopologyManagerPolicy))
+	rootCmd.PersistentFlags().BoolVar(&args.Info, "info", false, "Show cluster information, disregard the other arguments")
```

Comment:
Just one more comment that I missed in the review I submitted.
I think we should improve the message returned here. Instead of `disregard the other arguments` maybe we should say must specify  `--must-gather-dir-path`.

That makes me think of the case where the user provides arguments in addition to `--must-gather-dir-path`. Should we explicitly let the user know that other args are not needed or work as we do i.e. ignore the extra args and just show the cluster info. I am okay with the later but want to make sure we are consciously doing that and not because we missed this case.

Reply:
the standard practice is to always run the sanity checks on args, but just ignore the value which aren't needed on a given flow. This is also what we do atm. I don't think this behaviour is surprising or unfriendly. Are you thinking about specific flows?